File: SoloLearn Codes/Izzy the Iguana/izzy_the_iguana.py
dic = {
    "Mango": 9,
    "Carrot": 4,
    "Lettuce": 5,
    "Cheeseburger": 0
}

snacks = input().split()
print("Come on Down!" if sum(dic[i] for i in snacks) >= 10 else "Time to wait")

File: SoloLearn Codes/Izzy the Iguana/IzzyTheIguana.java
import java.util.Arrays;
import java.util.Scanner;
import java.util.Map;

public class IzzyTheIguana
{
    public static void main(String[] args) {
        String[] snacks = new Scanner(System.in).nextLine().split(" ");
        Map<String, Integer> snackList = Map.of("Lettuce", 5, "Carrot", 4, "Mango", 9, "Cheeseburger", 0);
        System.out.println(Arrays
                .stream(snacks)
                .mapToInt(c -> snackList.getOrDefault(c, 0))
                .sum() >= 10 ? "Come on Down!" : "Time to wait");
	}
}